### PR TITLE
Receiving translation for EU

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
EU translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_EU

### CODE REVIEW

File: eu/ja-jp/content/topics/end-user/device-access/macos-user-setup-password-sync.htm [hunk 1]
```diff
@@ -305,7 +305,7 @@ <h2>パスワードの再同期</h2>
                                                             <p>プロファイル画像の横の情報アイコンをクリックします。 </p>
                                                         </li>
                                                         <li value="3">
-                                                            <p>［Platform Single Sign-on（プラットフォームシングルサインオン）］の見出しの下で<span class="uicontrol">［Authenticate（認証）］</span>をクリックします。</p>
+                                                            <p>［Platform Single Sign-on heading（プラットフォームシングルサインオン）］の見出しの下で<span class="uicontrol">［認証］</span>をクリックします。</p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/device-access/macos-user-setup-password-sync.htm:308]: @pengyao-okta The first five files looked promising; the update missed here.

File: eu/ja-jp/content/topics/end-user/om-omm-user-android.htm [hunk 2]
```diff
@@ -316,15 +316,15 @@ <h4><a name="and_for_work"></a>Android for Workの登録</h4>
...
-                                                                <li value="6"><span class="uicontrol">［Accept（同意する）］</span>をタップしてGoogleのサービス利用規約に同意します。
-					<p>OMMデバイス登録が正常に完了します。Play for Workの登録が開始されます。数分かかることがあります。処理中にデバイスがロックされると、<span class="mc-variable okta-feature-names.Okta_Mobile variable">Okta Mobile</span>のPINの再入力が必要になります。</p><p>Play for Workにアクセスするには、左上隅にある<span class="uicontrol">メニュー</span>アイコンをタップします。次に、サイドバーの<span class="uicontrol">［Play for Work］</span>をタップします。</p><p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/end-user/omm-screenshot-android-11.png" data-mc-popup-alt="Successful enrollment confirmation page."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="1080" data-mc-height="1920" src="../../resources/images/end-user/omm-screenshot-android-11_thumb_220_220.png" alt="［Successful enrollment confirmation （登録成功の確認）］ページ。" tabindex="" /></a></p></li>
+                                                                <li value="6">[<span class="uicontrol">Accept (同意する)</span>]をタップしてGoogleのサービス利用規約に同意し、続行します。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/om-omm-user-android.htm:319]: Here, the brackets are outside the `<span>` elem

File: eu/ja-jp/content/topics/end-user/om-omm-user-android.htm [hunk 3]
```diff
@@ -316,15 +316,15 @@ <h4><a name="and_for_work"></a>Android for Workの登録</h4>
...
+					<p>OMMデバイス登録が正常に完了します。Play for Workの登録が開始されます。数分かかることがあります。処理中にデバイスがロックされると、<span class="mc-variable okta-feature-names.Okta_Mobile variable">Okta Mobile</span>のPINの再入力が必要になります。</p><p>Play for Workにアクセスするには、左上隅にある<span class="uicontrol">Menu（メニュー）</span>アイコンをタップします。次に、サイドバーの<span class="uicontrol">［Play for Work］</span>をタップします。</p><p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/end-user/omm-screenshot-android-11.png" data-mc-popup-alt="Successful enrollment confirmation page."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="1080" data-mc-height="1920" src="../../resources/images/end-user/omm-screenshot-android-11_thumb_220_220.png" alt="［Successful enrollment confirmation （登録成功の確認）］ページ。" tabindex="" /></a></p></li>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/om-omm-user-android.htm:320]: Here, the `<span>` elem for "Menu"  doesn't have any brackets

File: eu/ja-jp/content/topics/end-user/ov-device-health-trbl.htm [hunk 4]
```diff
@@ -269,10 +269,10 @@ <h1><span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta Fas
                                                     <h2>はじめに</h2>
                                                     <p>ご利用の<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>のバージョンが最新であることを確認します。 </p>
                                                     <h2>汎用メッセージ</h2>
-                                                    <p><span class="uicontrol">Device requirements not set（デバイス要件が設定されていません）</span>というメッセージが表示される場合、デバイスは組織のセキュリティ要件を満たしていません。<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>の<span class="wintitle">［デバイスの正常性］</span>ページで、アプリへのアクセスをブロックしているデバイスの状態を確認します。  </p>
+                                                    <p><span class="uicontrol">「Device requirements not set（デバイス要件が設定されていません）」</span>というメッセージが表示される場合、デバイスは組織のセキュリティ要件を満たしていません。<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>の<span class="wintitle">［デバイスの正常性］</span>ページで、アプリへのアクセスをブロックしているデバイスの状態を確認します。  </p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-device-health-trbl.htm:272]: Here we have the JA-style brackets within the `<span>` elem

File: eu/ja-jp/content/topics/end-user/ov-device-health-trbl.htm [hunk 5]
```diff
@@ -420,12 +420,12 @@ <h3><a name="Android"></a>Android </h3>
                                                                     <ol>
                                                                         <li value="1">
                                                                             <p>
-ご利用のデバイスで<span class="uicontrol">［Settings（設定）］</span>に移動します。
+ご利用のデバイスで<span class="uicontrol">［設定］</span>に移動します。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-device-health-trbl.htm:423]: @pengyao-okta One more monolingual label; the previous 37 files are clean, though, so the trend is still good.

File: eu/ja-jp/content/topics/end-user/ov-faq-ios.htm [hunk 6]
```diff
@@ -283,40 +281,37 @@ <h3><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verif
...
+                                                    <p>別のデバイスに<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>をインストールした場合、または電話（SMSまたは通話）、セキュリティキー、生体Authenticator、Google Authenticatorなど他の<span class="mc-variable okta-variables.ProductName variable">Okta</span>セキュリティ メソッドをセットアップした場合、これらのいずれかのAuthenticator方式を使って<span class="mc-variable okta-feature-names.Okta_End-User_Dashboard variable">Okta End-User Dashboard</span>にサインインしてください。お名前の横にある矢印をクリックし、[<span class="uicontrol">設定</span>]に移動します。下にスクロールして、［Extra Verification（追加検証）］または［Security Methods（セキュリティ メソッド）］に移動します。<span class="uicontrol">[Okta Verify]</span>を使用するには、<span class="uicontrol">[Setup（セットアップ）］</span>をクリックしてください。手順に従い、新しい<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>アカウントをデバイスに追加します。<span class="mc-variable okta-feature-names.Okta_End-User_Dashboard variable">Okta End-User Dashboard</span>にサインインできない場合は、Organizationのヘルプデスクにお問い合わせください。</p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-faq-ios.htm:309]: Interestingly, it's the same label across these files that is monolingual, "Settings (設定)".

File: eu/ja-jp/content/topics/end-user/ov-ios-add-acc-bluetooth.htm [hunk 7]
```diff
@@ -333,16 +323,10 @@ <h2>このタスクを開始する</h2>
                                                                                     <li value="2">
                                                                                         <p>
 
-<span class="uicontrol">［Add account to another device（別のデバイスにアカウントを追加する）］</span>をタップまたはクリックします。 
+<span class="uicontrol">［別のデバイスにアカウントを追加する］</span>をタップまたはクリックし、手順に従います。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-ios-add-acc-bluetooth.htm:326]: @pengyao-okta I see I jinxed it one more monolingual label

File: eu/ja-jp/content/topics/end-user/ov-ios-add-acc-bluetooth.htm [hunk 8]
```diff
@@ -355,33 +339,27 @@ <h2>このタスクを開始する</h2>
                                                             <p>新しいiOSデバイスに<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>をインストールします。</p>
                                                         </li>
                                                         <li value="3">
-                                                            <p>アプリを開き、<span class="uicontrol">［Add Account from Another Device（別のデバイスのアカウントを追加する）］</span>をタップします。</p>
+                                                            <p>アプリを開き、<span class="uicontrol">［別のデバイスのアカウントを追加する］</span>をタップします。</p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-ios-add-acc-bluetooth.htm:342]: and here

File: eu/ja-jp/content/topics/end-user/ov-ios-restore-same-device.htm [hunk 9]
```diff
@@ -263,52 +263,33 @@
...
-				<p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>の登録QRコードが画面に表示されます。</p></li></ol></li></ul></li>
+                                                        <li value="3"><span class="uicontrol">［プロファイルを編集する］</span>ボタンが表示されたらクリックし、プロンプトが表示されたらパスワードを入力します。  </li>
+                                                        <li value="4">新しいQRコードを取得します。下にスクロールして<span class="wintitle">［Extra Verification（追加検証）］</span>に移動します。組織によっては、このセクションを<span class="uicontrol">［セキュリティ方式］</span>と呼ぶ場合があります。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-ios-restore-same-device.htm:279]: Several monolingual labels in this file

File: eu/ja-jp/content/topics/end-user/ov-ios-trbl-push.htm [hunk 10]
```diff
@@ -263,25 +263,17 @@
...
                                                     <ol>
-                                                        <li value="1"><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、アカウントをタップします。</li>
-                                                        <li value="2">[Account Details（アカウントの詳細）]ページで、[Troubleshooting (トラブルシューティング)]のセクションにある[<span class="uicontrol">Push Notification Diagnostics (プッシュ通知診断)</span>]をタップします。診断ツールは、次のエリアを確認します。<ul><li>システム通知設定</li><li>プッシュトークン</li><li>通知サービス</li><li>デバイス時刻</li></ul></li>
+                                                        <li value="1"><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、自分のアカウントをタップします。</li>
+                                                        <li value="2"><span class="uicontrol">［アカウントの詳細］</span>ページの<span class="uicontrol">［トラブルシューティング］</span>セクションにある<span class="uicontrol">［プッシュ通知診断］</span>をタップします。診断ツールは、次のエリアを確認します。<ul><li>システム通知設定</li><li>プッシュトークン</li><li>通知サービス</li><li>デバイス時刻</li></ul></li>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-ios-trbl-push.htm:270]: monolingual labels in this file as well

File: eu/ja-jp/content/topics/end-user/ov-mng-account-ios.htm [hunk 11]
```diff
@@ -262,91 +262,59 @@
...
+                                                        <li value="5">アプリの手順に従ってアカウントのセットアップを完了します。 </li>
                                                     </ol>
                                                     <h3><a name="Reorder"></a>アカウントを並べ替える</h3>
                                                     <ol>
-                                                        <li value="1">デバイスで<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開きます。</li>
-                                                        <li value="2">メニュー (...) &gt; [<span class="uicontrol">編集</span>]をタップします。</li>
-                                                        <li value="3">アカウントの移動ハンドル<img src="../../resources/images/end-user/ov-ios-move-handle-acc_31x29.png" alt="iOSデバイスのOkta Verifyにあるアカウントの移動ハンドル" style="width: 31;height: 29;" />を長押しして、選択した場所にドラッグします。
+                                                        <li value="1"><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開きます。</li>
+                                                        <li value="2">メニュー （...） &gt; <span class="uicontrol">［Edit（編集）］</span>をタップします。</li>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-mng-account-ios.htm:281]: We have an issue in EN source, an unmarked UI control label in a click path

File: eu/ja-jp/content/topics/end-user/ov-mng-account-windows.htm [hunk 12]
```diff
@@ -265,27 +265,24 @@
...
+                                                    <h2><span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>認証のデフォルトアカウントを設定する</h2>
+                                                    <p>複数の<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>アカウントがあるときは、最も使用頻度の高いアカウントを<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>を使ったサインインのデフォルトアカウントとして設定できます。デフォルトアカウントは変更可能です。</p>
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、アカウントをクリックします。［<span class="uicontrol">Set as default for </span><span style="font-weight: bold;" class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>（Okta FastPassのデフォルトに設定する）］リンクをクリックします。<span class="wintitle">［Account Details（アカウントの詳細）］</span>ページの右上端に<span class="uicontrol">［</span><span style="font-weight: bold;" class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>のデフォルト］インジケーターが表示される場合、選択したアカウントはすでにデフォルトとして設定されています。 </p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-mng-account-windows.htm:278]: Something odd is happening here. The `<span>` elem for "Okta FastPass" is not formatted correctly, need to check EN to see where source of this error is

File: eu/ja-jp/content/topics/end-user/ov-mng-settings-ios.htm [hunk 13]
```diff
@@ -262,28 +262,28 @@
...
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>の詳細を確認するには、ドキュメントにアクセスします。<span class="uicontrol">［ヘルプ］</span>をタップします。</p>
                                                     <h3><a name="Send"></a>フィードバックを送信</h3>
-                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>について提案やアイデアを共有する、または発生する可能性のある問題を報告するには、右上端のメニュー（...）をタップし、<span class="menucascade"><span class="uicontrol">［Settings（設定）］</span><span class="uicontrol">［Send Feedback（フィードバックを送信）］</span></span>に移動します。アプリからフィードバックを送信するには、デバイスをシェイクします。 </p>
+                                                    <p><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>に関する提案やアイデアを共有したり、問題を報告したりするには、<span class="uicontrol">［フィードバックを送信］</span>をタップします。アプリからフィードバックを送信するには、デバイスをシェイクします。 </p>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-mng-settings-ios.htm:272]: monolingual here

File: eu/ja-jp/content/topics/end-user/ov-remediate-account-android.htm [hunk 14]
```diff
@@ -264,69 +264,42 @@
...
 </p>
-                                                    <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/end-user/ov-siw-acc-invalid.png" data-mc-popup-alt="Image of a warning that appears on the sign-in page if the Okta verify accout is invalid."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="411" data-mc-height="122" src="../../resources/images/end-user/ov-siw-acc-invalid_thumb_220_220.png" alt="Okta Verifyアカウントが無効になっている場合に、サインインページに表示される警告の画像です。" tabindex="" /></a>
+                                                    <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/end-user/ov-siw-acc-invalid.png" data-mc-popup-alt="Image of a warning that appears on the sign-in page if the Okta verify account is invalid."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="411" data-mc-height="122" src="../../resources/images/end-user/ov-siw-acc-invalid_thumb_220_220.png" alt="Okta Verifyアカウントが無効になっている場合に、サインインページに表示される警告の画像" tabindex="" /></a>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-remediate-account-android.htm:291]: We have image alt text here, it's in EN.

File: eu/ja-jp/content/topics/end-user/ov-send-logs-windows.htm [hunk 15]
```diff
@@ -265,27 +265,19 @@
...
-                                                        <p class="noteContent">使用できる<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>機能は、組織が行った構成によって異なります。</p>
+                                                        <p class="noteContent">管理者への注意：<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span> for Windowsは<a href="https://help.okta.com/okta_help.htm?type=oie&amp;id=ext-oie" target="_blank">Okta Identity Engine</a>のみで入手できます。</p>
                                                     </div>
                                                     <h2>このタスクを開始する</h2>
                                                     <ol>
-                                                        <li value="1">Windowsのシステムトレイで<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>アイコンを右クリックして［<span class="uicontrol">Report Issue（問題を報告する）</span>］を選択します。
+                                                        <li value="1">Windowsのシステムトレイで<span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>アイコンを右クリックして［<span class="uicontrol">問題を報告する</span>］を選択します。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-send-logs-windows.htm:272]: monolingual labels in this file

File: eu/ja-jp/content/topics/end-user/ov-set-default-fp-acc-android.htm [hunk 16]
```diff
@@ -263,22 +263,15 @@
...
+                                                        <li value="1"><span class="mc-variable okta-feature-names.Okta_Verify variable">Okta Verify</span>を開き、<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>を使ったサインインに使用するアカウントをタップします。</li>
+                                                        <li value="2"><span class="uicontrol">［アカウントの詳細］</span>ページの<span class="wintitle">［Options（オプション）］</span>に移動し、アカウントが<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>のデフォルトとしてすでに設定されていることを確認します。<ul><li>デフォルトの<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>アカウントになっていれば、緑色のチェックマークが表示されています。これでタスクは完了です。その他の操作はありません。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-set-default-fp-acc-android.htm:270]: monolingual labels in this file

File: eu/ja-jp/content/topics/end-user/ov-set-default-fp-acc-ios.htm [hunk 17]
```diff
@@ -263,22 +263,15 @@
...
-			<p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/end-user/ov-ios-default-fp-acc.png" data-mc-popup-alt="How to check if this is your default account for signing in with Okta FastPass"><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="595" data-mc-height="203" src="../../resources/images/end-user/ov-ios-default-fp-acc_thumb_220_220.png" alt="このアカウントがOkta FastPassでサインインするためのデフォルトのアカウントかを確認する方法" tabindex="" /></a></p></li><li>デフォルトの<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>アカウントではない場合、［<span class="uicontrol">Set as Default（デフォルトとして設定する）</span>］をタップします。</li></ul></li>
+                                                        <li value="2"><span class="uicontrol">［アカウントの詳細］</span>ページの<span class="wintitle">［Options（オプション）］</span>に移動し、アカウントが<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>のデフォルトとしてすでに設定されていることを確認します。<ul><li>デフォルトの<span class="mc-variable okta-feature-names.Okta_FastPass variable">Okta FastPass</span>アカウントになっていれば、緑色のチェックマークが表示されます。これでタスクは完了です。その他の操作はありません。
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-set-default-fp-acc-ios.htm:270]: monolingual labels in this file

File: eu/ja-jp/content/topics/end-user/ov-setup-ios.htm [hunk 18]
```diff
@@ -263,38 +263,24 @@
...
-                                                        </li>
-                                                        <li><a href="ov-new-install-signin-ios.htm" class="MCXref xref">サインインURLを使用してiOSデバイスにOkta Verifyをセットアップする</a>
-                                                        </li>
-                                                        <li><a href="ov-new-setup-no-qr-ios.htm" class="MCXref xref">アクティベーションリンクまたは秘密鍵を使ってiOSデバイスにOkta Verifyをセットアップする</a>
-                                                        </li>
-                                                        <li><a href="ov-ios-restore-same-device.htm" class="MCXref xref">同じiOSデバイスでOkta Verifyを復元する</a>
+                                                        <li><span class="uicontrol">コードを入力する</span>
                                                         </li>
-                                                        <li><a href="ov-ios-reset-register.htm" class="MCXref xref">Dashboardの設定ページから新しいiOSデバイスにOkta Verifyアカウントを追加する</a>
+                                                        <li><span class="uicontrol">プッシュ通知を受け取る</span>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/ov-setup-ios.htm:271]: monolingual labels in this file

File: eu/ja-jp/content/topics/end-user/plugin-switch-accounts.htm [hunk 19]
```diff
@@ -268,10 +268,10 @@ <h2>はじめに
...
-                                                        <li>Safari：ブラウザーツールバーの任意の場所を右クリックし、［<span class="uicontrol">Customize Toolbar（ツールバーをカスタマイズする）</span>］をクリックしてから、<span class="uicontrol">Your Apps（使用アプリ）</span>アイコンをブラウザーツールバーの希望する場所にドラッグします。</li>
+                                                        <li>Chrome：Chromeメニュー（<img src="../../resources/images/end-user/browser-menu-chrome.png" alt="Chromeメニュー" />）をクリックし、<span class="menucascade"><span class="uicontrol">［その他のツール］</span><span class="uicontrol">［拡張機能］</span></span>に移動して<span class="mc-variable okta-feature-names.Okta_Browser_Plugin variable">Okta Browser Plugin</span>が有効であることを確認します。プラグインが有効になっていることを確認したら、ブラウザーの右上部分にあるジグソーパズルアイコンをクリックし、<span class="mc-variable okta-feature-names.Okta_Browser_Plugin variable">Okta Browser Plugin</span>の横にあるピンアイコンを選択します。</li>
```
Comment (paulwallace-okta) [inline eu/ja-jp/content/topics/end-user/plugin-switch-accounts.htm:271]: monolingual labels in this file

**Review by paulwallace-okta — APPROVED** [review]

**Review by emikhasyak-okta — APPROVED** [review]
LGTM 🚢

**Pull request merged**